### ISSUE TITLE
1.0.0

### ISSUE THREAD

**Issue by rjsberry** [RazrFalcon/pico-args#61]
Have you considered releasing 1.0.0? If not, is there anything you have in mind that you'd like to add/change first?

`pico-args` is a really nice crate and seems to have been stable for a long time with very few issues raised against it.

**Comment by RazrFalcon** [comment]
Yes, probably a good idea. There are no future development planned.
